src/content/docs/en/recipes/i18n.mdx

```diff
@@ -0,0 +1,322 @@
+        - index.astro
+    </FileTree>
+
+    
+2. Set up `src/pages/index.astro` to redirect to your default language.
+
+    <StaticSsrTabs>
+      <Fragment slot="static">
+        ```astro
+        ---
+        // src/pages/index.astro
+        ---
+        <meta http-equiv="refresh" content="0;url=/en/" />
+        ```
+
+        This approach uses a [meta refresh](https://en.wikipedia.org/wiki/Meta_refresh) and will work however you deploy your site. Some static hosts also let you configure server redirects with a custom configuration file. See your deploy platform’s documentation for more details.
+      </Fragment>
+      
+      <Fragment slot ="ssr">
+        If you are using an SSR adapter, you can use [`Astro.redirect`](/en/guides/server-side-rendering/#astroredirect) to redirect to the default language on the server.
+
+        ```astro
+        ---
+        // src/pages/index.astro
+        return Astro.redirect('/en/');
+        ---
+        ```
+      </Fragment>
+    </StaticSsrTabs>
+
+### Use collections for translated content
+
+1. Create a folder in `src/content/` for each type of content you want to include and add subdirectories for each supported language. For example, to support English and French blog posts:
+
+    <FileTree>
+    - src/
+      - content/
+          - blog/
+            - **en/** Blog posts in English
+                - post-1.md
+                - post-2.md
+            - **fr/** Blog posts in French
+              - post-1.md
+              - post-2.md
+    </FileTree>
+
+2. Create a `src/content/config.ts` file and export a collection for each type of content.
+
+    ```ts
+    //src/content/config.ts
+    import { defineCollection, z } from 'astro:content';
+
+    const blogCollection = defineCollection({
+      schema: z.object({
+        title: z.string(),
+        author: z.string(),
+        date: z.date()
+      })
+    });
+
+    export const collections = {
+      'blog': blogCollection
+    };
+
+    ```
+    📚 Read more about [Content Collections](/en/guides/content-collections/).
+
+3. Use [dynamic routes](/en/core-concepts/routing/#dynamic-routes) to fetch and render content based on a `lang` and a `slug` parameter.
+
+    <StaticSsrTabs>
+      <Fragment slot="static">
+        In static rendering mode, use `getStaticPaths` to map each content entry to a page:
+
+        ```astro
+        //src/pages/[lang]/blog/[...slug].astro
+        ---
+        import { getCollection } from 'astro:content'
+
+        export async function getStaticPaths() {
+          const pages = await getCollection('blog')
+
+          const paths = pages.map(page => {
+            const [lang, ...slug] = page.slug.split('/');
+            return { params: { lang, slug: slug.join('/') || undefined }, props: page }
+          })
+
+          return paths;
+        }
+
+        const { lang, slug } = Astro.params;
+        const page = Astro.props;
+        const formattedDate = page.data.date.toLocaleString(lang);
```

Comment:
`toLocaleString()` could potentially be confusing for those not familiar with it. Maybe a callout of what that is and link to some nice MDN docs?